### PR TITLE
Visit Table Loader

### PR DESCRIPTION
**Pull request by Johngtka**
Features List:
- [X] `added loader to visit table`

Bez tytułu

### CODE REVIEW

**Comment by Johngtka** [comment]
S&M commit proposal:
`feat: loader for visit table (#36)`

File: src/app/visit/visit.component.css [hunk 1]
```diff
@@ -37,7 +37,19 @@
...
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
 .selectedRow,
 selectedRow:hover {
     background-color: lightgray !important;
-}
+}
```
Comment (macrog) [inline src/app/visit/visit.component.css:55]: what do you have set that always remove the last empty line?

File: src/app/visit/visit.component.html [hunk 2]
```diff
@@ -71,41 +64,25 @@
...
-        (click)="createVisitPointsTable()"
-        mat-raised-button
-        color="primary"
-    >
+    <button class="add-btn" [hidden]="!showCheck || showFinish" (click)="createVisitPointsTable()" mat-raised-button
+        color="primary">
         {{ 'PATIENT_VISIT.FINISH_BUTTONS.CHECK' | translate }}
     </button>
-</div>
+</div>
```
Comment (macrog) [inline src/app/visit/visit.component.html:88]: empty line

**Review by macrog — CHANGES_REQUESTED** [review]

File: src/app/visit/visit.component.html (hunk 2 above)
Comment (Johngtka) [reply src/app/visit/visit.component.html]: done

File: src/app/visit/visit.component.css (hunk 1 above)
Comment (Johngtka) [reply src/app/visit/visit.component.css]: done

File: src/app/visit/visit.component.css [hunk 3]
```diff
@@ -37,6 +37,18 @@
...
+    min-height: 200px;
+    max-height: 400px;
+    overflow: auto;
+    background: rgba(0, 0, 0, 0.15);
+    height: 30vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
```
Comment (macrog) [inline src/app/visit/visit.component.css:51]: duplicated class with styles, move it to the `styles.css`

**Review by macrog — CHANGES_REQUESTED** [review]

**Pull request merged**